weblate/glossary/tasks.py

```diff
@@ -28,6 +28,29 @@ def sync_glossary_languages(pk: int, component: Component | None = None):
     )
     if not missing:
         return
+
+    # Identify unnecessary empty glossary languages
```

Comment:
This would be executed only in case some languages are missing, see the above condition.